src/appleseed.studio/mainwindow/project/projectmanager.cpp

```diff
@@ -89,8 +91,14 @@ void ProjectManager::load_project_async(const string& filepath)
 
     m_async_io_filepath = QString::fromStdString(filepath);
 
+#if defined(__GNUC__) && (__GNUC__ > 4)
```

Comment:
This code has been there for ages, did it not compile with gcc 4.8?

Reply:
It does not compile when using Qt 5.12.

Reply:
It does not compile when using Qt 5.12. We cannot include QtConcurrent when using gcc 4.8.

Reply:
Does not compile when using Qt 5.12. Since this is the only problem, I tried a quick workaround.

Reply:
Ok.